rest_framework/utils/field_mapping.py

```diff
@@ -203,6 +203,9 @@ def get_relation_kwargs(field_name, relation_info):
     if to_many:
         kwargs['many'] = True
 
+    if to_field:
+        kwargs['to_field'] = to_field
```

Comment:
I named it to_field because mentally i am still treating it like a ForeignKey object at that point. Pointing it to a slug_field afterwards was just handy